app/serializers/user_custom_fields_serializer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# A basic user serializer, with custom fields
+class UserCustomFieldsSerializer < BasicUserSerializer
```

Comment:
One thing that's confusing here is we already have a model called `UserCustomField` and that means this would be the default serializer for that model.

Could we use a name like `UserWithCustomFields` instead to differentiate?

Reply:
Oh, that's a good one - can do! Naming is hard.